src/Builtins.jl

```diff
@@ -170,19 +171,52 @@ function show(io::IO, ::MIME"text/html", select::Select)
     withtag(io, :select) do
         for o in select.options
             print(io, """<option value="$(htmlesc(o.first))"$(select.default === o.first ? " selected" : "")>""")
-            withtag(io, :option, :value=>o.first) do
-                if showable(MIME"text/html"(), o.second)
-                    show(io, MIME"text/html"(), o.second)
-                else
-                    print(io, o.second)
-                end
+            if showable(MIME"text/html"(), o.second)
+                show(io, MIME"text/html"(), o.second)
+            else
+                print(io, o.second)
+            end
+            print(io, "</option>")
+        end
+    end
+end
+
+get(select::AbstractSelect) = ismissing(select.default) ? first(select.options).first : select.default
```

Comment:
this is not the default value for MultiSelect, so let's just make them two separate types without a supertype?

The default value should be 
```julia
get(select::MultiSelect) = ismissing(select.default) ? Any[] : select.default
```

also see my comment about the `default`

Reply:
You are right, I'll correct this.